src/Mod/Draft/Draft.py

```diff
@@ -6133,17 +6133,15 @@ def execute(self, obj):
             ff8 = obj.FontFile.encode('utf8')                  # 1947 accents in filepath
                                                                # TODO: change for Py3?? bytes?
                                                                # Part.makeWireString uses FontFile as char* string
-#            CharList = Part.makeWireString(obj.String,obj.FontFile,obj.Size,obj.Tracking)
-            CharList = Part.makeWireString(obj.String,ff8,obj.Size,obj.Tracking)
+            CharList = Part.makeWireString(obj.String,obj.FontFile,obj.Size,obj.Tracking)
```

Comment:
This might not make it work for py2 anymore (the encoding will still be needed). So I'd prefer to put this case inside an `if sys.version_info.major < 3: `...

Reply:
okay, changed that way